parm/atm/obs/config/sondes.yaml

```diff
@@ -0,0 +1,309 @@
+obs space:
+  name: sondes
+  obsdatain:
+    obsfile: $(OBS_DIR)/$(OBS_PREFIX).sondes.$(OBS_DATE).nc4
+    obsgrouping:
+      group variables: ["station_id", "LaunchTime"]
```

Comment:
Is LaunchTime now in the obs files?   It was absent in early tests.

Reply:
That depends on the source of the observations. Which is something we should probably think about. If using GSI ncdiags, to get LaunchTime, the GSI observer has to be re-run. Otherwise, we can just grab the operational diags and convert them.

Reply:
Thanks for the explanation.